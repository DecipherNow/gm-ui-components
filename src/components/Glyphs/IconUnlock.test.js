import React from "react";
import { shallow } from "enzyme";

import IconUnlock from "./IconUnlock";

describe("IconUnlock", () => {
  it("matches snapshot", () => {
    const aIconUnlock = shallow(<IconUnlock />);
    expect(aIconUnlock).toMatchSnapshot();
  });
});