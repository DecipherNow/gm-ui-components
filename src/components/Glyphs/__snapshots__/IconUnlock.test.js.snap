// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconUnlock matches snapshot 1`] = `<IconMock />`;
